modules/ROOT/pages/microsoft/microsoft-powershell-connector-examples.adoc

```diff
@@ -0,0 +1,262 @@
+= Microsoft Windows PowerShell Connector Examples
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
```

Comment:
For the template, I would like to see the examples named and some sort of boilerplate snippet here about how the audience should best use these examples.